test/js/bun/util/CPUFeatures.test.ts

```diff
@@ -0,0 +1,21 @@
+import { getCPUFeatures, type X86CPUFeature, type AArch64CPUFeature } from "bun:internal-for-testing";
+
+import { test, expect } from "bun:test";
+
+test("CPUFeatures", () => {
+  const features = getCPUFeatures();
+  console.log({ features });
+
+  if (process.arch === "x64") {
+    const x86 = features as X86CPUFeature;
+    expect(x86, "CI must have AVX enabled").toHaveProperty("avx", true);
+    expect(x86, "CI must have AVX2 enabled").toHaveProperty("avx2", true);
+    // sanity check:
+    expect(x86.sse42).toBeTrue();
```

Comment:
If there are other CPU features we always expect to be true, we might as well check them here.